ForkClone.md

```diff
@@ -1,33 +1,33 @@
-# Fork and Clone 
+# Fork and clone 
```

Comment:
Because the rest of the headings used sentence case, I changed this one to sentence case. If you prefer to leave them all as title case, that's fine, too. I look for consistency throughout the doc.